uint64 register
uint64 value
---
string return_value
